internal/kms/kmip.go

```diff
@@ -313,9 +313,6 @@ func (kms *kmipKMS) getSecrets() (map[string]string, error) {
 		switch k {
 		case kmipClientKey, kmipCLientCert, kmipCACert, kmipUniqueIdentifier:
 			config[k] = string(v)
-		default:
```

Comment:
I dont see any reason to make kmip as special here. This default check exists for all other kms as well. If we allow adding more data to the secret specific to csi there are chances for human mistakes when they update/recreate the secrets. why can't admin create an extra secrete specific to csi itself without any unwanted data?

Reply:
btw admin can create separate Secrets, I am just confirming if we should or not...

Reply:
it's good to have a dedicated secret for the below reasons
* To avoid human errors 
* Keeps the secret data very minimal and even more secured if it is compromised as we are going to use secret per service it's not going to affect all the services.
* Avoid multiple services depending on one global secret

Reply:
ack ! thanks !